OpenDreamRuntime/DreamRuntime.cs

```diff
@@ -157,6 +157,7 @@ public class DreamRuntime
                 int elapsedTime = (int)(new DateTimeOffset(DateTime.Now).ToUnixTimeMilliseconds() - TickStartTime);
                 int tickLength = (int)(100 * WorldInstance.GetVariable("tick_lag").GetValueAsFloat());
                 int timeToSleep = tickLength - elapsedTime;
+                WorldInstance.SetVariable("cpu", new DreamValue(100 - timeToSleep));
```

Comment:
This assumes a `world.tick_lag` of 1. `world.cpu` should be a ratio of `elapsedTime` and `tickLength`.
```suggestion
                WorldInstance.SetVariable("cpu", new DreamValue((float)elapsedTime / tickLength * 100));
```